app/celery/scheduled_tasks.py

```diff
@@ -385,24 +387,26 @@ def delete_old_records_from_events_table():
 
 @notify_celery.task(name="zendesk-new-email-branding-report")
 def zendesk_new_email_branding_report():
-    previous_weekday = datetime.today().date() - timedelta(days=1)
+    previous_bst_weekday = convert_utc_to_bst(datetime.utcnow()).date() - timedelta(days=1)
```

Comment:
Don't think this is the best name, because it's wrong half of the year when we're not in BST.

I think `previous_weekday` is fine. If we need to add a comment saying that we're checking for daylight savings let's do that.

Reply:
hmmm we do have a habit (of referring to `Europe/London` as "bst". See `bst_date` column in ft_notification_status, and indeed the `convert_utc_to_bst`. Perhaps bst isn't the best word in conjunction with date (rather than datetime) but i'll have a think.

Reply:
If we do it elsewhere, then ok.

Weird tho.